### DISCUSSION TITLE
DatabaseSessionHandler::addUserInformation?

### DISCUSSION THREAD

**Discussion by rmcdaniel** [laravel/framework#38335] [Q&A]
https://github.com/laravel/framework/blob/8.x/src/Illuminate/Session/DatabaseSessionHandler.php#L197

Why does the database driver add the user_id in this manner while the redis/cookie/file/etc drivers do not?

**Comment by tpetry** [comment]
The attribute is for e.g. jetstream to be able to search for all sessions of a user. But redis/cookies/files do not have indexes like a database to efficiently replicate the search functionality. So there is no need for it.
